resources/dicts/patrols/general/medcat.json

```diff
@@ -1069,5 +1069,48 @@
     "max_cats": 2,
     "antagonize_text": null,
     "antagonize_fail_text": null
+},
+{
+    "patrol_id": "gen_med_discussion1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["med_cat", "random_herbs", "respect", "trust", "med_only", "no_app"],
+    "intro_text": "p_l and r_c decide to look for herbs whilst discussing the latest news.",
+    "decline_text": "It seems like some mischievous cat has gotten into the herb storage and they have to sort it all over again.",
+    "chance_of_success": 50,
+    "exp": 10,
+    "success_text": [
+    "They have a good discussion while foraging for herbs and manage to come up with several ideas to discuss with the leader and deputy.",
```

Comment:
hmmm i'm not sure the 'come up with several ideas' concept of this outcome make much sense with the context of the intro?  Perhaps something more geared towards the 'latest news' that was first mentioned would be better.  Or giving further context within this outcome text as to what those ideas are for.  Maybe the latest news had to do with med cats in a different clan? or a sickness in a different clan?